allensdk/internal/README.md

```diff
@@ -11,4 +11,35 @@ The dependencies specific to this subpackage are listed in internal_requirements
     make install
     ```
 - mpi4py : we conda install this like `conda install mpi4py`
-- opencv : we conda install this like `conda install -c conda-forge opencv`
\ No newline at end of file
+- opencv : we conda install this like `conda install -c conda-forge opencv`
+
+Accessing Databases
+===================
+If you want to access the on-prem databases using the AllenSDK without providing credentials every time,
+you'll need to export the following environment variables. We recommend adding the following
+to your ~/.bash_profile on Linux or macOS machines (empty quotes are strings you need to fill in):
+
+If you do not know the credentials and need access, contact [Rob Young](RobY@alleninstitute.org) or [Wayne Wakeman](waynew@alleninstitute.org).
```

Comment:
this prob. ought to be a shared mailbox / address

Reply:
This is what Wayne signed off on initially. I'll also be sending an email to all the internal users with instructions.